src/common/client-app.ts

```diff
@@ -69,8 +74,8 @@ export class App {
 
   onInterval= () => {
     var current = this.boardVM.toPlain();
-    var myChanges = rfc6902.createPatch(this.shadow, current);
-    if (myChanges.length) {
+    var myChanges = rfc6902.createPatch(this.shadow, current);
+    if (myChanges.length) {
```

Comment:
You're right, changes must be applied on client's shadow too when a change is sended to server.